jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/feedback/SnackbarTests.java

```diff
@@ -1,43 +1,187 @@
+        put("topCenter", "TopCenter");
+        put("topRight", "TopRight");
+        put("bottomRight", "BottomRight");
+        put("bottomCenter", "BottomCenter");
+        put("bottomLeft", "BottomLeft");
+        put("topLeft", "TopLeft");
+    }};
+    private static String cssConstanValue = "MuiSnackbar-root MuiSnackbar-anchorOrigin";
+
+    @BeforeMethod
+    public void before() {
+        openSection("Snackbar");
+    }
+
     @Test
     public void simpleSnackbarTest() {
-        feedbackSnackbarSimplePage.open();
 
-        simpleSnackbarFrame.snackbar.is().notExist();
-        simpleSnackbarFrame.openSnackbarButton.click();
-        simpleSnackbarFrame.snackbar.is().visible();
-        simpleSnackbarFrame.snackbar.has().content("Note archived");
-        simpleSnackbarFrame.snackbar.getActionByIndex(1).has().text("UNDO");
-        simpleSnackbarFrame.snackbar.clickActionByIndex(2);
-        simpleSnackbarFrame.snackbar.is().notExist();
+        simpleSnackbarPopUp.is().notVisible();
+        simpleSnackbarButton.click();
+        simpleSnackbarPopUp.is().text("Note archived");
+        undoSnackbarButton.is().text("UNDO");
+        undoSnackbarButton.click();
+        simpleSnackbarPopUp.is().notVisible();
+        simpleSnackbarButton.click();
+        closePopUpSnackbarButton.click();
+        simpleSnackbarPopUp.is().notVisible();
+    }
+
+    @Test
+    public void customizedSnackbarTest() {
+
+        customizedSnackbarPopUpTitles.get(1).is().text("This is an error message!");
+        customizedSnackbarPopUpStyles.get(1).has().classValue(String.format("MuiPaper-root MuiAlert-root MuiAlert-filled%s MuiPaper-elevation6", "Error"));
+        customizedSnackbarPopUpTitles.get(2).is().text("This is a warning message!");
+        customizedSnackbarPopUpStyles.get(2).has().classValue(String.format("MuiPaper-root MuiAlert-root MuiAlert-filled%s MuiPaper-elevation6", "Warning"));
+        customizedSnackbarPopUpTitles.get(3).is().text("This is an information message!");
+        customizedSnackbarPopUpStyles.get(3).has().classValue(String.format("MuiPaper-root MuiAlert-root MuiAlert-filled%s MuiPaper-elevation6", "Info"));
+        customizedSnackbarPopUpTitles.get(4).is().text("This is a success message!");
+        customizedSnackbarPopUpStyles.get(4).has().classValue(String.format("MuiPaper-root MuiAlert-root MuiAlert-filled%s MuiPaper-elevation6", "Success"));
+
+        successSnackbarPopUp.is().notVisible();
+        successSnackbarButton.click();
+        successSnackbarPopUp.is().text("This is a success message!");
+        closePopUpSuccessButton.click();
+        successSnackbarPopUp.is().notVisible();
+    }
+
+    @Test
+    public void positionedSnackbarTest() {
+
+        positionedSnackbarButtons.get(1).click();
+        snackbarPopUpText.is().text("I love snacks");
+        positionedSnackbarPopUp.has().classValue(String.format("%s%s", cssConstanValue, map.get("topCenter")));
```

Comment:
Isn't it enough to check topCenter without cssConstanValue?
Why we need map? Isn't it enough to use value text itself?